### ISSUE TITLE
Integration of contacts to report

### ISSUE THREAD

**Issue by ghost** [NREL/NEB-Tool#125]

**Comment by koay9f** [comment]
Contacts are connected to Assessments directly and via systems/end uses/KPMs. 
In Reports (maybe custom or a header that is "Contact Reports")
Create a report for a specific person
It would have 
* Exe summaries of all assessments that they are connected to
* KPM data
     *  If connected to a KPM, and that KPM is connected to a NEB within an assessment - something like the potential impact table in the KPM + a bit more context?
     *  
Willing to accept other ideas

**Comment by RLiNREL** [comment]
Since there may be many contacts and some of them don't have connections with assessments/others, a separate contact report type may not make sense. Instead, add a section of contacts in each individual assessment report would be more interesting to users I think: it shows what stakeholders may be involved with this particular assessment.

For example, we can have a table, two columns, one is the different type, the other is the stakeholder name (contact detail can be viewed in a pop up)

- Overall Assessment (directed tagged to the assessment): Sam, Sarah
- EEM (connected through the linked inventory and end uses)
  - EEM 1: Sam
  - EEM 2: Sarah
  - ..
- NEB (directly tagged to neb)
  - NEB1: Sarah
  - NEB2 ..

**Comment by koay9f** [comment]
I agree with the idea of a "connections section" in the report - even expanding it beyond stake holders - some kind of recreation of the side bar connections result, but that would look better on a page than a narrow portion of the window.

But this report was intended for the point where you might have several projects in JUSTIFI and want to give Quinn Quality a quick report of "these are all the things we found that will impact your area". It isn't intended to be a report for ALL stakeholders, but a filtered report for one stakeholder at a time. And very short, The Exe. Summary plus a bit more details.

**Comment by RLiNREL** [comment]
Thanks! I see the intended use case now. Then, we could do something like the custom report that allow the user to select a contact from a list and then a executive summary style report will be rendered for that person.
